src/coreclr/src/ToolBox/SOS/DacTableGen/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
should this be 3.6.2 as well here?

Reply:
actually I see a few other files that use 3.6.3 instead, we should be consistent.

Reply:
This particular project (which is Windows only) needs CMake 3.8's C# support. This check ensures we don't try building it on platforms with 3.6 only.